bio/minibwa/map/wrapper.py

```
@@ -0,0 +1,154 @@
+                parse_output_format=True,
+            )
+            # Pipe the output to samtools view to enable CRAM compression, or
+            # plain SAM output.
+            out_cmd += f"  --out '/dev/stdout' "
+            out_cmd += f" | samtools view {samtools_view_opts}"
+        else:
+            # Else, let sambamba write bam.
+            out_cmd += f"  --out '{snakemake.output[0]}' "
+
+            # Optionally write index
+            if "idx" in snakemake.output.keys():
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)